rotkehlchen/tests/api/test_users.py

```diff
@@ -1022,7 +1081,8 @@ def test_users_by_name_endpoint_errors(rotkehlchen_api_server, username, db_pass
     assert rotki.user_is_logged_in is False
 
     # Now let's try to login with an invalid password
-    data = {'password': 'wrong-password', 'sync_approval': 'unknown', 'async_query': True}
+    data = {'password': 'wrong-password', 'sync_approval': 'unknown', 'async_query': True}  # type: ignore  # TODO: mypy for some reason says that this statement is unreachable. Not sure why.
```

Comment:
same here. And I suspect the reason is always the same. Something to do with `assert rotki.user_is_logged_in is False` right above